src/RawModelExportList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import Button from '@mui/material/Button'
+import Divider from '@mui/material/Divider'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/system/Box'
+import VisibilityIcon from '@mui/icons-material/VisibilityTwoTone'
+
+import { useGetModelById, useGetModelVersions } from '../data/model'
+import { Deployment } from '../types/interfaces'
+
+const DeploymentVersion = ({ text, type }: { text: string; type: string }) => {
+  const [displayText, setDisplayText] = React.useState<boolean>(false)
+
+  const toggleText = () => {
+    setDisplayText(!displayText)
+  }
+
+  return (
+    <Stack direction='row'>
+      <Button sx={{ m: 1 }} variant='text' onClick={toggleText} startIcon={<VisibilityIcon />}>
+        {displayText ? 'Hide' : 'Show'} {type} path
+      </Button>
+      <Box component={Stack} direction='column' justifyContent='center'>
+        <Typography variant='body1'>{displayText ? text : 'xxxxxxxx/xxxxxxxx/xxxxxxxx'}</Typography>
```

Comment:
I think it'd be better to hide the implementational detail of their location all together and just have two buttons for "code" and "binary".  They're not sensitive, they're entirely predictable based on deployment ID.  People don't really need to know where they are, just have the ability to copy them / visit them.

Reply:
<img width="364" alt="Screenshot 2022-08-15 at 00 01 16" src="https://user-images.githubusercontent.com/89583054/184558039-81763623-3ef2-4a63-9c17-d57bdf6287c2.png">

What about this type of screen with an input box + copy button?

Reply:
Yeah so the reason I've hidden them isn't really out of any sort of sensitivity, it was more to make the page look less jarring because of all the text on it.

I can try the above - I'll see how it looks!